test/helpers/backend/land_parcels_helper_test.rb

```diff
@@ -2,6 +2,32 @@
 
 module Backend
   class LandParcelsHelperTest < ActionView::TestCase
-    # Add tests here...
+    class Bidule
```

Comment:
:smile: Maybe rename 'Bidule' (and following variables like _truc, etc.) ?

Reply:
Did it, you can check it !